api/swagger.yaml

```diff
@@ -8876,13 +8876,21 @@ paths:
                   SubnetSize specifies the subnet size of the networks created from the default subnet pool
                 type: "integer"
                 format: "uint32"
+              DataPathPort:
+                description: |
+                  DataPathPort specifies the data path port number for data traffic.
+                  Acceptable port range is 1024 to 49151.
+                  if no port is set or os set to 0, default port 4789 will be used.
+                type: "integer"
```

Comment:
Actually, looks like swagger allows specifying a minimum and maximum https://swagger.io/docs/specification/data-models/data-types/

Not sure if that works well for us, because `0` is also valid, so lets keep that for a follow-up to try